src/qt/listctrl.cpp

```diff
@@ -888,16 +912,12 @@ bool wxListCtrl::SortItems(wxListCtrlCompare WXUNUSED(fn), wxIntPtr WXUNUSED(dat
 }
 
 // ----------------------------------------------------------------------------
-// virtual list controls (not currently implemented in wxQT)
+// virtual list controls ( OnGetItemColumnImage not currently implemented in wxQT)
 // ----------------------------------------------------------------------------
 
-wxString wxListCtrl::OnGetItemText(long WXUNUSED(item), long WXUNUSED(col)) const
+wxString wxListCtrl::OnGetItemText(long item, long col) const
 {
-    // this is a pure virtual function, in fact - which is not really pure
-    // because the controls which are not virtual don't need to implement it
-    wxFAIL_MSG( wxT("wxListCtrl::OnGetItemText not supposed to be called") );
-
-    return wxEmptyString;
+    return GetItemText(item, col);
```

Comment:
No, this is wrong. `OnGetItemText()` is _called_ by wxWidgets, it's not supposed to be implemented by it. I.e. the program can define the text of the item.